featurebyte/query_graph/node/metadata/sdk_code.py

```diff
@@ -74,6 +74,12 @@ def as_input(self) -> str:
         return str(self)
 
 
+class InfoStr(str):
+    """
+    InfoStr class is used to represent a string that contains information about the code.
```

Comment:
Does `InfoStr` has to be a string? It seems responsible for passing on some key value information for subsequent node's code generation, rather than being converted to code directly by itself. What do you think about having this class handle the to and from json? Is that conversion to json still required if this is not a str?

Reply:
This is a good point. The initial reason to inherit as a string is for the experimentation of the fix. Will change the type to something like `InfoDict` which inherit dictionary. Doing do, we can avoid unnecessary conversion between string & dict.